### PR TITLE
option to pass in multiple urls to good to go check

### PR DESCRIPTION
**Pull request by wheresrhys**
@matthew-andrews

### CODE REVIEW

File: bin/next-build-tools.js [hunk 1]
```diff
@@ -41,11 +41,13 @@ program
 	.option('-s, --skip-gtg', 'skip the good-to-go HTTP check')
 	.option('--skip-enable-preboot', 'skip the preboot')
 	.option('--docker', 'deploy an app which uses docker')
+	.option('--gtg-urls <urls>', 'Comma separated list of urls to check before concluding the app is ok (these are in addition to __gtg)', list)
```
Comment (matthew-andrews) [inline bin/next-build-tools.js]: i really like commander.  stuff like this so neat.

File: tasks/wait-for-gtg.js [hunk 2]
```diff
@@ -2,31 +2,38 @@
...
-				});
-		}
-		checker = setInterval(checkGtg, 3000);
-		timeout = setTimeout(function() {
-			console.log("2 minutes passed, bailing");
-			reject(name + '.herokuapp.com/__gtg not responding with an ok response within 2 minutes');
-			clearInterval(checker);
-		}, 2*60*1000);
-	});
+	var urls = ['__gtg'];
```
Comment (matthew-andrews) [inline tasks/wait-for-gtg.js]: do you think it's better to always have `/__gtg` or if the developer is overriding the default make them take responsibility for providing them?  doesn't really matter i guess.

**Comment by matthew-andrews** [comment]
nice :+1:

**Comment by wheresrhys** [comment]
Boom 

```
polling: http://ft-article-branch-236.herokuapp.com/__gtg
polling: http://ft-article-branch-236.herokuapp.com/d0d1932a-3b05-11e5-8613-07d16aad2152
polling: http://ft-article-branch-236.herokuapp.com/__gtg
polling: http://ft-article-branch-236.herokuapp.com/d0d1932a-3b05-11e5-8613-07d16aad2152
polling: http://ft-article-branch-236.herokuapp.com/__gtg
polling: http://ft-article-branch-236.herokuapp.com/d0d1932a-3b05-11e5-8613-07d16aad2152
polling: http://ft-article-branch-236.herokuapp.com/__gtg
polling: http://ft-article-branch-236.herokuapp.com/d0d1932a-3b05-11e5-8613-07d16aad2152
poll __gtg ok
```

File: tasks/wait-for-gtg.js (hunk 2 above)
Comment (wheresrhys) [inline tasks/wait-for-gtg.js]: related question, should next-express provide __gtg?

**Comment by matthew-andrews** [comment]
does it wait for all them of them to be ok?

File: tasks/wait-for-gtg.js (hunk 2 above)
Comment (matthew-andrews) [inline tasks/wait-for-gtg.js]: no it should not imho

Comment (matthew-andrews) [inline tasks/wait-for-gtg.js]: we force it to be implemented by having the check in nbt deploy and hopefully in the future __gtg will be smarter.

**Pull request merged**